pkg/minikube/reason/reason.go

```diff
@@ -173,8 +173,9 @@ var (
 		ID:       "RSRC_DOCKER_STORAGE",
 		ExitCode: ExInsufficientStorage,
 		Advice: `Try one or more of the following to free up space on the device:
-	
-			1. Run "docker system prune" to remove unused Docker data (optionally with "-a")
+			1. Remove unused Docker data (optionally with "-a") 
```

Comment:
Should this should say "Remove Docker daemon generated outside of minikube"?

Reply:
I am not sure if talking about outside and inside minikbue would make sense for the average end user. That might confuse them even more.
Since we give them the shell command with $ ... to copy paste.

If we wanted inside minikube we would give them this command
$ minikbue ssh docker system prune ...

Reply:
By outside of minikube, I mean images that were not created by minikube, as this command explicitly filters those out. I'm not sure of the exact best verbiage here, but the comment is no longer accurate to the command it describes.